### PR TITLE
sample updates

### PR DESCRIPTION
**Pull request by lsmallwood**
@ldanzinger - please review

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Routing/ServiceArea/README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ Barriers can also be added which can affect the impedance by not letting traffic
 ## How to use sample
 
 - In order to find any services areas at least one `ServiceAreaFaciltiy` needs to be added.
-- To add a facility, click the Facility button then click anywhere on the `MapView`.
-- To add a barrier, click the Barrier button and click multiple locations on sMapViews to draw a barrier.
+- To add a facility, select "Facility" from the combo-box then click anywhere on the `MapView`.the `
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Routing/ServiceArea/README.md]: remove "the `"

File: ArcGISRuntimeSDKQt_QMLSamples/Routing/ServiceArea/Info.plist [hunk 2]
```diff
@@ -3,9 +3,9 @@
 <plist version="1.0">
 <dict>
 	<key>CFBundleDisplayName</key>
-	<string>SrviceArea</string>
+  <string>ServiceArea</string>
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Routing/ServiceArea/Info.plist]: indentation is off

**Comment by lsmallwood** [comment]
@ldanzinger - changes pushed please merge

**Comment by ldanzinger** [comment]
looks good

**Pull request merged**